core/src/test/java/org/elasticsearch/search/sort/FieldSortBuilderTests.java

```diff
@@ -25,10 +25,14 @@
 
     @Override
     protected FieldSortBuilder createTestItem() {
+        return randomFieldSortBuilder();
+    }
+
+    public static FieldSortBuilder randomFieldSortBuilder() {
         String fieldName = rarely() ? SortParseElement.DOC_FIELD_NAME : randomAsciiOfLengthBetween(1, 10);
         FieldSortBuilder builder = new FieldSortBuilder(fieldName);
         if (randomBoolean()) {
-            builder.order(RandomSortDataGenerator.order(builder.order()));
+            builder.order(RandomSortDataGenerator.order(null));
```

Comment:
So - by default, order is initialised to ASC.

Before this change, calling this method would always set the order DESC, so as we are calling it randomly, half of all tests run with the original ASC, half run with DESC.

After this change, calling this method will always set the order to ASC, so all tests should be running with ASC sorting, leaving DESC untested.

So - why this change?

If this was unintentional, most likely "RandomSortDataGenerator" is a bad name for a class that returns a random value except the one provided.